en/docs/observe-and-analyze/observe/observability-overview.md

```diff
@@ -11,13 +11,17 @@ The following diagram shows the Observability dashboard:
 
 ![Dashboard overview](../../assets/img/observability/overview-overall.png){.cInlineImage-full}
 
+!!! note
+    The observability dashboard for non-Ballerina components do not display the low-code diagram. Therefore, the functionality within the low-code diagram is limited only to Ballerina components. 
```

Comment:
1. Change "do not" to "does not".
2. The second sentence can give the wrong idea (that limitations apply only to Ballerina component). Shall we change it to "Therefore, the functionality within the low-code diagram is available only for Ballerina components."?

Reply:
done